src/tracks/Pattern.cpp

```diff
@@ -65,6 +65,26 @@ Pattern::Pattern( InstrumentTrack * _instrument_track ) :
 	m_steps( MidiTime::stepsPerTact() )
 {
 	setName( _instrument_track->name() );
+
+	// Resize this track to be the same as existing tracks in the BB
+	const TrackContainer::TrackList & tracks =
+		m_instrumentTrack->trackContainer()->tracks();
+	for(unsigned int trackID = 0; trackID < tracks.size(); ++trackID)
+	{
+		if(tracks.at(trackID)->type() == Track::InstrumentTrack &&
+			tracks.at(trackID) != m_instrumentTrack)
+		{
+			int currentTCO = m_instrumentTrack->getTCOs().indexOf(this);
```

Comment:
When limiting the line to 80 characters, remember that one tab equals to 8 spaces.